DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -1110,7 +1110,9 @@ internal async Task OnGuildMemberUpdateEventAsync(TransportMember member, Discor
             var pending_old = mbr.IsPending;
             var roles_old = new ReadOnlyCollection<DiscordRole>(new List<DiscordRole>(mbr.Roles));
             var avatar_old = mbr.GuildAvatarHash;
+            var username_old = mbr.Username;
 
+            mbr.Username = member.User.Username;
```

Comment:
`DiscordMember#Username` is just a passthrough to the user object if I’m not mistaken? Using that would be more consistent.